packages/cli/README.md

```diff
@@ -273,6 +273,30 @@ bubblewrap fingerprint generateAssetLinks <flags>
 Flags:
  - `--output=<filename>`: path from where to load the project configuration.
 
+## `play`
+
+:warning: This is an experimental feature.
+
+Manages the artifacts for your Google Play Project.
+
+Usage:
+
+```
+bubblewrap play --serivceAccountFile="/path/to/serivce/account"  --retain 86 --publish
```

Comment:
nit: typo on service (twice).

Reply:
Could this also be `/path/to/service/account.json`?

Reply:
Done